app/controllers/shared/api.php

```diff
@@ -281,7 +281,7 @@
             throw new Exception(Exception::USER_PASSWORD_RESET_REQUIRED);
         }
 
-        if ($mode !== APP_MODE_ADMIN) {
+        if ($mode !== APP_MODE_ADMIN && !Auth::isPrivilegedUser(Authorization::getRoles())) {
```

Comment:
Let's remove this since it's going to be removed by https://github.com/appwrite/appwrite/pull/7984

Reply:
removed